buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -279,6 +279,48 @@ protected ByteBuf newDirectBuffer(int initialCapacity, int maxCapacity) {
 
         return toLeakAwareBuffer(buf);
     }
+    /**
+     *  Default number of heap areanas - System Property: io.netty.allocator.numHeapArenas - default 2 * cores
+     */
+    public static int defaultNumHeapArena() {
```

Comment:
I see now that we are exposing other internal details of this class (`smallCacheSize`, etc...) so nvm.